.php_cs

```diff
@@ -0,0 +1,80 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->in(__DIR__);
+
+$fixers = array(
```

Comment:
Reviewed the PR and looking good! Mostly it is formatting/style corrections. However seems these are not necessarily PSR2 corrections, but nonetheless good. Let me merge it this weekend, as I want to check a few little things more